Error compiling since 0.9.2

Hi,

I just updated to 0.9.2 and I am having the following error when compiling.

`android\src\main\java\com\RNFetchBlob\RNFetchBlobBody.java:107: error: method contentLength() is already defined in class RNFetchBlobBody
    public long contentLength() {
                ^
Note: Some input files use or override a deprecated API.`

_There's duplicated code in the RNFetchBlobBody.java at lines 97 and 107_

--- a/src/android/src/main/java/com/RNFetchBlob/RNFetchBlobBody.java
+++ b/src/android/src/main/java/com/RNFetchBlob/RNFetchBlobBody.java
@@ -104,11 +104,6 @@ public class RNFetchBlobBody extends RequestBody{
     }
     
     @Override
-    public long contentLength() {
-        return contentLength;
-    }
-    
-    @Override
     public void writeTo(BufferedSink sink) {
         try {
             if (requestType == RNFetchBlobReq.RequestType.AsIs)

